website/src/app/blog/may-2024-update/readme.mdx

```diff
@@ -1,270 +1,88 @@
-plane responds with a healthy Gateway to connect to. If there are multiple
-healthy Gateways, the control plane will round-robin between them, effectively
-splitting the load across all Gateways in the Site.
-
-Need more throughput? Simple: deploy more Gateways. The control plane will
-automatically distribute the load across all of them.
-
-We think high availability is such a core feature in a remote access solution
-that we made failover and load balancing available **on all plans**, including
-the Starter tier. [Read more](/kb/deploy/gateways) about how it works in our
-documentation.
-
-#### Firewall hole-punching
-
-You know what's not fun? Configuring firewalls.
+<Image
+  src="/images/blog/may-2024-update/traffic-restrictions.png"
+  alt="Firezone traffic restrictions"
+  width={600}
+  height={600}
+  className="mx-auto"
+/>
 
-More precisely, configuring your organization's cloud or corporate firewalls to
-allow incoming connections from the internet. Not only is it a pain to manage at
-scale, it also exposes your organization to all kinds of security risks.
+Simply add the desired filters when creating or editing a Resource, and only
+that type of traffic will be allowed. This is useful for allowing access to some
+services running on a host but not all of them.
 
-So we rearchitected Firezone to include the same NAT traversal techniques that
-WebRTC applications have enjoyed for years now:
-[STUN](https://www.rfc-editor.org/rfc/rfc8489.html) and
-[TURN](https://www.rfc-editor.org/rfc/rfc8553), known collectively as
-[ICE](https://datatracker.ietf.org/doc/html/rfc8445).
+#### How it works
 
-As you can probably surmise from the above links, these are well-established
-standards for doing reliable NAT traversal. These have been battle-tested in the
-field for years across all kinds of products -- Firezone is only the latest to
-benefit from them.
+When you add traffic restrictions to a Resource, all Gateways that can serve
+that Resource will immediately receive the updated Resource configuration.
 
-What does this mean for you? It means you can deploy Firezone without touching a
-single firewall configuration and still enjoy the same level of performance as
-if you did. Attack surface is minimized and connections are direct. It's a
-win-win.
+Restrictions are enforced at the **Gateway** level. If a user tries to access a
+service on the Resource not allowed by the restriction, the Gateway will simply
+drop the traffic, preventing it from ever reaching the Resource in question.
 
-For the curious readers, you can find our implementation of ICE, aptly named
-"snownet", in our repository
-[here](https://github.com/firezone/firezone/tree/main/rust/connlib/snownet).
+This is useful, for example, to allow access to some services running on a host
+(like SSH) to your DevOps team for maintenance purposes, while only allowing
+HTTP and HTTPS traffic to the host for the rest of your organization.
```

Comment:
I don't think I'm following here.  We don't associate ports/protocols with policies right?  How are we able to prevent a group from accessing a specific port once there is a policy that gives that group access to the resource?

Reply:
Yeah let me reword it. You would create two Resources.

Reply:
I think this can still go out given the solution to our problem noted in #4939. Anyone that manages to hit this issue will be resolved in that issue.

Reply:
Sounds good 👍